newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -1249,7 +1277,7 @@ const VariablesList = (props: Props) => {
         }
       });
     },
-    []
+    [setSelectedNodes]
```

Comment:
It's essential that all these callbacks stay the same even when selectedNodes change. Otherwise, the editor has to re-render TONS of stuff at every single change.
That's why "setSelectedNodes" must be stable even when the selection change.